fig.yml

```diff
@@ -0,0 +1,18 @@
+galera1:
+  build: .
+  command: --wsrep-cluster-address=gcomm://
```

Comment:
The trick here is that the peers tell each other their ip addresses and there is no concept of a leader and followers. The first peer in the cluster only opens the "session" and the others need one address of a peer to join it. As soon as others have joined the cluster the peers distribute information among them automatically.